### PR TITLE
spinner on sass created

### PR DESCRIPTION
**Pull request by StasDomanskyi**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -52,6 +52,12 @@
                 </div>
             </div>
         </header>
+        <section class="section">
+            <div class="spinner">
+                <div class="spinner__triangle"></div>
+                <div class="spinner__inner--round"></div>
```
Comment (LenaAviles) [inline index.html]: why do you need modifier `--round`? do you have anywhere just `spinner__inner` element?

File: sass/variables.sass [hunk 2]
```diff
@@ -1,5 +1,38 @@
 $yellow: #ffdd57
 
+.spinner
```
Comment (LenaAviles) [inline sass/variables.sass]: why is your spinner styles lives in `variables.sass`?

File: index.html (hunk 1 above)
Comment (StasDomanskyi) [reply index.html]: okay, we have fixed excessive modifier

File: sass/variables.sass (hunk 2 above)
Comment (StasDomanskyi) [reply sass/variables.sass]: okay, we have created another file main.sass for all future styles and spinner

**Pull request merged**